.github/dependabot.yml

```diff
@@ -9,4 +9,11 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+    open-pull-requests-limit: 1
+  - package-ecosystem: "nuget"
+    directory: "/src/Authentication/Authentication.Test"
```

Comment:
Just to confirm here. We don't want this for all projects?

Reply:
No, we don't need for all projects because;
1. Moq is only used in Authentication module.
2. Blanket upgrade of all other dependencies might break experience especially for users using PowerShell 5. It needs to be managed i.e. Every upgrade to be treated individually with justification. For example, the changes here are due to a security compliance issue